frontend/src/app/(routes)/multisig/components/multisig-account/Transactions.tsx

```
@@ -172,21 +172,21 @@ const TransactionsFilters = ({
-    txnsCount && txnsCount.forEach((t: any) => {
-      if (t?.computed_status?.toLowerCase() === option.toLowerCase()) {
-        count = t?.count
-      }
-    })
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    txnsCount &&
+      txnsCount.forEach((t: any) => {
+        if (t?.computed_status?.toLowerCase() === option.toLowerCase()) {
+          count = t?.count;
+        }
+      });
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)